report/src/statChart.ts

```diff
@@ -0,0 +1,183 @@
+        if (_ === void 0) return this._mean;
+        this._mean = _;
+        return this;
+    }
+
+    private _stdDev: number;
+    standardDeviation(): number;
+    standardDeviation(_: number): this;
+    standardDeviation(_?: number): number | this {
+        if (_ === void 0) return this._stdDev;
+        this._stdDev = _;
+        return this;
+    }
+
+    private _quartiles: QuertilesType = [0, 0, 0, 0, 0];
+    quartiles(): QuertilesType;
+    quartiles(_: QuertilesType): this;
+    quartiles(_?: QuertilesType): QuertilesType | this {
+        if (_ === void 0) return this._quartiles;
+        this._quartiles = _;
+        return this;
+    }
+
+    private stdDev(degrees: number): number {
+        return this._mean + degrees * this._stdDev;
+    }
+
+    private formatDev(degrees: number): string {
+        return siFormatter(this.stdDev(degrees));
+    }
+
+    refreshData() {
+        const stdDevs: { label: string, value: number }[] = [
+            { label: "", value: this.stdDev(-4) },
+            { label: this.formatDev(-3), value: this.stdDev(-3) },
+            { label: "-2σ", value: this.stdDev(-2) },
+            { label: "-1σ", value: this.stdDev(-1) },
+            { label: this.formatDev(0), value: this.stdDev(0) },
+            { label: "+1σ", value: this.stdDev(1) },
+            { label: "+2σ", value: this.stdDev(2) },
+            { label: this.formatDev(3), value: this.stdDev(3) },
+            { label: "", value: this.stdDev(4) }
+        ]
+
+        const domainMin = this._quartiles[0];
+        const domainMax = this._quartiles[4];
+        const padding = (domainMax - domainMin) / 20;
+        const stdData = stdDevs.filter(sd => sd.value >= domainMin && sd.value <= domainMax);
+        const ticks = [{ label: siFormatter(domainMin), value: domainMin.toString() }, ...stdData.map(sd => { return { label: sd.label, value: sd.value.toString() }; }), { label: siFormatter(domainMax), value: domainMax.toString() }];
+        this.domainAxis.ticks(ticks);
+        this
+            .xAxisDomainLow(domainMin - padding)
+            .xAxisDomainHigh(domainMax + padding)
+            .data([
+                [this.stdDev(-4), 0],
+                [this.stdDev(-3), 0.3],
+                [this.stdDev(-2), 5],
+                [this.stdDev(-1), 68],
+                [this.stdDev(0), 100],
+                [this.stdDev(1), 68],
+                [this.stdDev(2), 5],
+                [this.stdDev(3), 0.3],
+                [this.stdDev(4), 0]
+            ])
+            ;
+    }
+
+    update(domNode, element) {
+        this.refreshData();
+        super.update(domNode, element);
+    }
+}
+
+export class StatChart extends HTMLWidget {
+
+    _bellCurveDiv: any;
+    _candleDiv: any;
+
+    _bellCurve = new BellCurve()
+        .columns(["", "Std. Dev."])
+        .paletteID("Quartile")
+        .interpolate_default("basis")
+        .pointSize(0)
+        .xAxisType("linear")
+        .xAxisOverlapMode("none")
```

Comment:
Maybe "stagger" to fix the squashed graph issue Dan pointed out?

Reply:
Tried it an it looks worse...